instances/azp-build-asg/instances.tf

```diff
@@ -32,7 +32,7 @@ data "template_file" "init" {
 resource "aws_launch_template" "build_pool" {
   name_prefix   = "${var.ami_prefix}_${var.azp_pool_name}"
   image_id      = data.aws_ami.azp_ci_image.id
-  instance_type = var.instance_type
+  instance_type = var.instance_types[0]
```

Comment:
As a side note, it's not that big of a deal now, cause newer terraform has dynamic, but it would've been significantly less lines to just create a launch template per instance type.

Reply:
I'll follow up on this later.